src/inference/include/openvino/runtime/properties.hpp

```diff
@@ -373,6 +373,11 @@ static constexpr Property<bool> enable_profiling{"PERF_COUNT"};
  */
 namespace log {
 
+#ifdef DEBUG
+#    define WAS_DEBUG DEBUG
```

Comment:
=> _OV_WAS_DEBUG? to minimize conflicts with users code

Reply:
Changed